### PR TITLE
Updates for January Release

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by asimons-mule — COMMENTED** [review]
1) We are still filtering by environments, tags and services (not dependencies), so we shouldn't change the language there. 
2) The "Show dependents and dependencies" is a toggle (_note: we've changed from a check box_). [This allows you to choose whether or not you want to see the dependencies of the nodes/services with the selected tags (the nodes related to it) in addition to the services that have the selected tags (automatically in the list). The list is still of services, however. Some are just dependencies of others.]
screen shot 2019-01-02 at 4 36 03 pm

**Review by asimons-mule — APPROVED** [review]
Double word in this sentence: "This feature enables you to hide the nodes that you do not want to ### see see in the graph."
Otherwise - looks good!

File: modules/ROOT/pages/view.adoc [hunk 1]
```diff
@@ -72,11 +72,11 @@ From the side panel, you can rename the connection by clicking the *Set Edge Nam
 [[customize-view]]
 == Customize Your View of Your Network
 
-Use the filtering feature in the side panel to customize your view and choose what appears in the canvas. You can filter by *environments*, *tags*, and *dependencies*.
+Use the filtering feature in the side panel to customize your view and choose what appears in the canvas. You can filter by *environments*, *tags*, and *services*.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/view.adoc]: Don't bold unless UI for translation

**Review by Mulesoft-Mysti — APPROVED** [review]
Approve with the one suggested change.

File: modules/ROOT/pages/view.adoc [hunk 2]
```diff
@@ -59,9 +59,9 @@ When you select a node within the graph, the node and its dependency stack will
 * *Metrics chart*: Shows the time-series chart of the selected metric in the *Metric* drop-down.
 * *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can change the layer and the tags from this section. See xref:layers.adoc[Use Layers in Anypoint Visualizer] and xref:use-tags-in-visualizer.adoc[Use Tags in Anypoint Visualizer].
 
-From the side panel, you can rename a node by clicking the *Rename Node* button.
+From the side panel, you can click *Rename Node* to rename a node.
 
-You can search for a node (service) in the *Search services* text field. Services containing your search in any part of their name will be highlighted, while others will fade.
+You can search for a node (service) in the *Search services* text field. Services containing your search in any part of their name will be highlighted, while others fade.
```
Comment (george-hoffer) [inline modules/ROOT/pages/view.adoc]: Please change "will be highlighted" to "are highlighted"

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**